src/extension/index.ts

```diff
@@ -112,24 +105,15 @@ export async function activate(context: vscode.ExtensionContext) {
   context.subscriptions.push(walletDetector);
 
   context.subscriptions.push(
-    vscode.window.registerTreeDataProvider(
-      "neo3-visual-devtracker.views.blockchains",
-      blockchainsTreeDataProvider
-    )
+    vscode.window.registerTreeDataProvider("neo3-visual-devtracker.views.blockchains", blockchainsTreeDataProvider)
```

Comment:
Why did we change the formatting for these subscriptions?

Reply:
I couldn't figure out the previous formatted setting, so this is automatically formatted with 120 column width. The current formatting setting is out of the box from the prettier extension (which is the same extension used previously). So instead of reversing engining the format, we can just use this going forward.